README.md

```diff
@@ -53,7 +54,8 @@ rails s
 - [GitHub](https://github.com/elassis)
 - [LinkedIn](https://linkedin.com/in/enmanuel-lassis-pena)
 
-## 🧠 UI Designer Credit 📝
+## 🧠 Acknowledgements 📝
+### Credits to the UI designer 
 - [Gregoire Vella](https://www.behance.net/gregoirevella)
```

Comment:
- [ ] OPTIONAL
The _acknowledgment_ sub header is occurring twice in your ReadMe. Kindly ensure to improve upon this aspect

![image](https://user-images.githubusercontent.com/12745474/153308962-7b641158-7f81-40c0-8c95-f756541f35b5.png)

Reply:
thanks i will fix it up